src/clr-angular/timeline/_timeline.clarity.scss

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('timeline.clarity') {
```

Comment:
I don't see any dark theme styles here, has this been considered or spec'd out?

Reply:
Unfortunately it was never in the spec.